### ISSUE TITLE
Teams search does not include content on Sharepoint pages/news added as tabs in a Team

### ISSUE THREAD

**Issue by jeffmparker** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#6329]
Hi, I use a Sharepoint site page in my Teams as a team Home page as a tab, and also a separate News feed tab. However, the Teams search function does not include content on these pages.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5346de0f-d09a-86ce-8390-5a33d8139bfb
* Version Independent ID: 5052193b-fd71-d1fe-e64c-f2c47218531c
* Content: [Use Content Search in Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/content-search)
* Content Source: [Teams/content-search.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/content-search.md)
* Service: **msteams**
* GitHub Login: @markjjo
* Microsoft Alias: **markjjo**

**Comment by markjjo** [comment]
@jeffmparker It's unclear what you mean by "Teams search function" and how you're configuring the search. In the content search tool, you can search all sites in your org (include sites associated with a Team) by turning on the sites toggle on the "Modify locations" page when you're configuring a search. Alternatively, you can try to use the "path:<home site URL>" or "documentlink:<home site URL> property:value pair in the keywords box. Not sure what/if news feeds items are searchable. Otherwise, would suggest you open a ticket with MS Support.

**Comment by jeffmparker** [comment]
Hi, thanks, I mean from the search box at the top of the desktop app. When I do this, content from SPO pages that are accessible within Teams is not displayed.

I don’t see an option to “modify locations” in search. The app says I have Microsoft Teams Version 1.3.00.30866 (64 bit).

**Comment by markjjo** [comment]
@jeffmparker The article that you logged this github issue against about using the content search eDiscovery tool to search for Teams content. This is an admin tool that is located in the Microsoft 365 compliance center that requires that you're assigned permissions to access/use it. That's what I assume you were referring and therefore my reference to "modifying" the content locations that you are searching. So I completely misunderstood your question/problem, which seems to be about using the search box in the Teams client. Unfortunately, I can't help with that and don't know if it's possible to search the tabs you're referring to.

I did a quick search and found this topic about using the client to search Teams: https://support.microsoft.com/en-us/office/search-for-messages-and-more-in-teams-4a351520-33f4-42ab-a5ee-5fc0ab88b263 Though you may have already found this.

**Comment by jeffmparker** [comment]
Thanks @markjjo. I see now that the page I asked about is indeed in the compliance area. I stumbled upon it google searching the problem I described. Perhaps the page could benefit from something at the top that highlights that its content relates to compliance, eDiscovery, etc. and not generally about searching in Teams, which is what I thought it did.

Besides this though, my problem still exists, and I'm wondering where/how it might be most suitably raised?
